features.json

```diff
@@ -103,36 +168,50 @@
 			"logo": "/images/chrome.svg",
 			"features": {
 				"bigInt": "85",
+				"branchHinting": ["flag", "Requires CLI flag `--js-flags=--experimental-wasm-branch-hinting`"],
 				"bulkMemory": "75",
+				"compilationHints": ["flag", "Requires CLI flag `--js-flags=--experimental-wasm-compilation-hints`"],
+				"customAnnotationSyntaxInTheTextFormat": null,
 				"exceptions": "95",
 				"extendedConst": "114",
 				"gc": "119",
+				"instrumentAndTracingTechnology": ["flag", "Requires CLI flag `--js-flags=--experimental-wasm-instruction-tracing`"],
 				"jspi": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-jspi`"],
+				"jsStringBuiltins": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
 				"memory64": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
 				"multiMemory": "120",
 				"multiValue": "85",
 				"mutableGlobals": "74",
-				"relaxedSimd": "114",
+				"referenceTypedStrings": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
 				"referenceTypes": "96",
+				"relaxedSimd": "114",
 				"saturatedFloatToInt": "75",
+				"sharedEverythingThreads": ["flag", "Requires CLI flag `--js-flags=--experimental-wasm-shared-everything-threads`"],
 				"signExtensions": "74",
 				"simd": "91",
+				"stackSwitching": ["flag", "Requires CLI flag `--js-flags=--experimental-wasm-stack-switching`"],
 				"tailCall": "112",
 				"threads": "74",
-				"typeReflection": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"]
+				"typedFunctionReferences": "119",
+				"typeReflection": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
+				"webContentSecurityPolicy": "97"
 			}
 		},
 		"Firefox": {
 			"url": "https://www.mozilla.org/firefox/",
 			"logo": "/images/firefox.svg",
 			"features": {
 				"bigInt": "78",
+				"branchHinting": ["flag", "Enabled in Nightly, unavailable in Beta/Release"],
 				"bulkMemory": "79",
+				"customAnnotationSyntaxInTheTextFormat": null,
 				"exceptions": "100",
 				"extendedConst": "112",
 				"gc": "120",
 				"jspi": ["flag", "Requires flag `javascript.options.wasm_js_promise_integration` in Nightly, unavailable in Beta/Release"],
+				"jsStringBuiltins": ["flag", "Enabled in Nightly, requires flag in Beta/Release"],
```

Comment:
This one is weird, it's somehow also available on Beta / Release:  https://bugzilla.mozilla.org/show_bug.cgi?id=1883921

Reply:
I updated it, and some others with the name of the flags.